UI/Selenium/Pages/CameraWorkingPage.cs

```diff
@@ -0,0 +1,20 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UISelenium.Pages
+{
+    public class CameraWorkingPage
+    {
+
```

Comment:
please remove unnecessary blank lines to avoid number of lines

Reply:
Removed